### PR TITLE
Cleanup of 'Regression' and 'MathUtilities' classes

### PR DESCRIPTION
**Pull request by BrianCollinss**
Working on #9349 

I will be happy to revert any chance that seems undesirable.

### CODE REVIEW

File: Models/Graph/Regression.cs [hunk 1]
```diff
@@ -237,7 +237,7 @@ public IEnumerable<IAnnotation> GetAnnotations()
                     StringBuilder text = new StringBuilder();
                     text.AppendLine($"y = {stats[i].Slope:F2}x + {stats[i].Intercept:F2}, r\u00B2 = {stats[i].R2:F2}, n = {stats[i].n:F0}");
                     text.AppendLine($"NSE = {stats[i].NSE:F2}, ME = {stats[i].ME:F2}, MAE = {stats[i].MAE:F2}");
-                    text.AppendLine($"RSR = {stats[i].RSR:F2}, RMSD = {stats[i].RMSE:F2}");
+                    text.AppendLine($"RMSE = {stats[i].RMSE:F2}, RMR = {stats[i].RMR:F2}");
```
Comment (hol353) [inline Models/Graph/Regression.cs:240]: Happy to have RMSD renamed RMSE (we've been meaning to do that for a while) and have RMR added. Can you add RSR back onto graph annotations. Lots of users expect it.

**Comment by BrianCollinss** [comment]
Thanks @hol353. Would you also comment on 'removing the spaces around equal signs'. It saves some space, though some people may find it cluttered.

image
image
image

**Comment by hol353** [comment]
Yes, good idea. I like a more compact stats area on graphs. Go and remove the spaces.

**Comment by BrianCollinss** [comment]
Final product so far:

image
image

**Comment by hol353** [comment]
Looks good to me. Let me know when you're done and I'll merge.

**Comment by BrianCollinss** [comment]
Please go ahead and merge. Ta

**Pull request merged**